packages/solarwinds-apm/src/config.ts

```diff
@@ -188,13 +203,14 @@ export interface Config extends z.input<typeof schema> {
 /** Processed configuration for solarwinds-apm */
 export interface Configuration extends z.output<typeof schema> {
   service: string
+  appoptics: boolean
   legacy: boolean
 
+  headers: Record<string, string>
   otlp: {
     tracesEndpoint?: string
     metricsEndpoint?: string
     logsEndpoint?: string
-    headers: Record<string, string>
   }
```

Comment:
`headers` field moved out of `otlp` since they're now also used by the sampler !